src/ring_theory/multiplicity.lean

```diff
@@ -260,6 +260,23 @@ begin
   rwa zero_dvd_iff,
 end
 
+lemma multiplicity_eq_multiplicity_mk [decidable_rel ((∣) : associates α → associates α → Prop)]
+  {a b : α} : multiplicity a b = multiplicity (associates.mk a) (associates.mk b) :=
```

Comment:
Typically we put the more complicated expressions on the left of an equality sign:
```suggestion
lemma multiplicity_mk_eq_multiplicity [decidable_rel ((∣) : associates α → associates α → Prop)]
  {a b : α} : multiplicity (associates.mk a) (associates.mk b) = multiplicity a b :=
```